Files/UserControls/LayoutModes/GenericFileBrowser.xaml

```diff
@@ -192,6 +192,16 @@
                     <FontIcon Glyph="&#xE7F4;" />
                 </MenuFlyoutItem.Icon>
             </MenuFlyoutItem>
+            <MenuFlyoutItem
+                x:Name="SetAsLockscreenBackgroundItem"
+                x:Uid="BaseLayoutItemContextFlyoutSetAsLockscreenBackground"
```

Comment:
Have you added a string to the en-US resource file?

Reply:
Sorry, I forgot about that. I am adding that now.

Reply:
for MultilingualResources too

Reply:
MultilingualResources will be updated automatically when we build the application before an update. There isn't a need to hold back on merging because of that.